sandy/core/endf6.py

```diff
@@ -266,8 +270,16 @@ def get_endf6_file(library, kind, zam, to_file=False):
         text = "".join([foo_read(name, url) for name in files.values()])
         tape = Endf6.from_text(text)
     else:
-        filename = files[zam]
-        tape = foo_get(filename, url)
+        if type(zam) == list:
```

Comment:
use `isinstance`.
I don't like this...what if instead of a list you get an array, or a tuple...it will break!

Check if `hasattr(zam, "__len__")` to see if it is an iterable

Reply:
I thought that with isinstance will not break because i introduce more ways to group data